src/components/checkboxes/_checkboxes.scss

```diff
@@ -100,4 +100,20 @@
   .govuk-c-checkboxes__input:disabled + .govuk-c-checkboxes__label {
     opacity: .5;
   }
+
+  .govuk-c-checkboxes__conditional {
+    margin-left: $govuk-spacing-scale-3;
+    @include govuk-responsive-margin($govuk-spacing-responsive-4, "bottom");
+    padding-left: 28px;
+    border-left: $govuk-border-width-mobile solid $govuk-border-colour;
+
+    @include mq($from: tablet) {
+      margin-left: 18px;
+      padding-left: 33px;
+    }
+
+    & [class^="govuk-c-"]:last-child {
+      margin-bottom: 0;
+    }
```

Comment:
I wonder if the number of elements we want to apply this to is definite eg. `govuk-c-label`, `govuk-c-input` - the wildcard here feels like it could throw up unexpected results.

Reply:
Not really no, we know they will be components but could be anything. Input, text area, another fieldset etc 

I'm not sure the results would be unexpected, it will remove the margin-bottom of the last govuk component inside a conditional wrapper. If it did it on some and not others I'd be concerned that would be more unexpected?

Reply:
Thanks @dashouse, that sounds okay to me.

Let me know when this is ready to review, I think the input `name` flagged below is the only thing outstanding.